features/gui/admin_panel/player_manager.lua

```diff
@@ -0,0 +1,323 @@
+local Actions = require 'features.gui.admin_panel.functions'.actions
+local AdminPanel = require 'features.gui.admin_panel.core'
+local Global = require 'utils.global'
+local Gui = require 'utils.gui'
+local Table = require 'utils.table'
+
+local main_button_name = Gui.uid_name()
+local selection_switch_name = Gui.uid_name()
+local selection_dropdown_name = Gui.uid_name()
+
+local on_cheat_mode = Gui.uid_name()
+local on_show_reports = Gui.uid_name()
+local on_create_pool = Gui.uid_name()
+local on_revive_ghosts = Gui.uid_name()
+local on_save_game = Gui.uid_name()
+local on_delete_blueprints = Gui.uid_name()
+local on_destroy_speakers = Gui.uid_name()
+local on_remove_biters = Gui.uid_name()
+local on_remove_enemies = Gui.uid_name()
+local on_add_regular = Gui.uid_name()
+local on_add_probation = Gui.uid_name()
+local on_jail_player = Gui.uid_name()
+local on_remove_regular = Gui.uid_name()
+local on_remove_probation = Gui.uid_name()
+local on_unjail_player = Gui.uid_name()
+local on_invoke_player = Gui.uid_name()
+local on_goto_player = Gui.uid_name()
+local on_spank_player = Gui.uid_name()
+local on_ban_player = Gui.uid_name()
+-- local on_teleport = Gui.uid_name()
+-- local on_destroy_selected = Gui.uid_name()
+
+local this = {
+  ---@type table<number, table< index: number, name: string >>
+  player_selection = {},
+  ---@type table<number, string>
+  selection_switch = {},
+  ---@type table<number, table<string>>
+  ban_items = {},
```

Comment:
```suggestion
  player_ban_items = {},
```
A bit confusing that this has the same name as `local ban_items` below.